### ISSUE TITLE
C.saveConfig Error on MATLAB 2019a Linux

### ISSUE THREAD

**Issue by yg-smile** [mathworks-ref-arch/matlab-apache-spark#1]
Dear authors,

When I tried to run `C.saveConfig` (in MATLAB 2019a Linux), it complained that `Error: Functions cannot be indexed using {} or . indexing`. Could you please help me? Thank you!

Here is a screenshot:

![image](https://user-images.githubusercontent.com/49927412/188734396-cd420d96-084e-45e7-a40f-90d568a78671.png)

**Comment by asollander** [comment]
Hi, 

thanks for reaching out to us. I looked into this, and it turns out this kind of dot indexing was introduced in R2019b (see doc [here](https://www.mathworks.com/help/matlab/release-notes.html#mw_efbf4fe4-e1da-46f1-af3d-5a75a422a678)).

We will at the fix in the next release. For the time being, you can edit the `pretty` file yourself, and change the line to

```matlab
prettyPrinter = com.google.gson.GsonBuilder().setPrettyPrinting().create();
```

Best regards,
Anders

**Comment by asollander** [comment]
This is now available in release v0.1.19, released today.

**Comment by yg-smile** [comment]
> 

Thank you @asollander , I can use the dot indexing `C.saveConfig` now.
